Part2_Mapplots/ex2_7_subplots.ipynb

```diff
@@ -106,19 +83,19 @@
     "\n",
     "There are some functions in pyplot that can potentially solve this problem. (However, they don't).\n",
     "\n",
-    "* `f.tight_layout()`, which tries to optimise the room used for the subplots.\n",
+    "* `f.tight_layout()`, which tries to optimize the room used for the subplots.\n",
     "* `subplot_adjust`, i.e. setting the distance between the subplots manually\n",
     "\n",
     "Before we can illustrate these two, we need to get another obstacle out of the way.\n",
     "\n",
     "\n",
     "## `bbox_inches='tight'`\n",
     "\n",
-    "When you save a figure you can tell matplotlib to remove all white boundary areas using `plt.savefig('figure.pdf', bbox_inches='tight')`. While this sounds like a good idea, the issue is that the final figure size is quite unpredictable. This can distort your font sizes. Also, if a journal asks for a figure with a certain size you cannot use this option.\n",
+    "When you save a figure you can tell matplotlib to remove all white boundary areas using `plt.savefig('figure.pdf', bbox_inches='tight')`. While this sounds like a good idea, the issue is that the final figure size is unpredictable. This can distort your font sizes. Also, if a journal asks for a figure with a certain size you cannot use this option.\n",
```

Comment:
```suggestion
    "When you save a figure you can tell matplotlib to remove all white boundary areas using `plt.savefig('figure.pdf', bbox_inches='tight')`. While this sounds like a good idea, the issue is that the size of the figure is unpredictable. This can distort your font sizes. Also, if a journal asks for a figure with a certain size you cannot use this option.\n",
```